code/game/objects/items/toys/toys.dm

```diff
@@ -504,3 +504,30 @@
 							/obj/item/toy/prize/odysseus					= 1,
 							/obj/item/toy/prize/phazon						= 1
 							)
+
+/obj/item/toy/festivizer
```

Comment:
Where's the can be held by xenos flag

Reply:
there isn't one but thats what this proc is for

![image](https://user-images.githubusercontent.com/53100513/208535832-b7be4e0c-9005-4d10-a19f-e372d58c2844.png)